code/modules/reagents/Chemistry-Reagents/Medicine-Reagents.dm

```diff
@@ -173,7 +173,7 @@
 		if(method == INGEST)
 			M.adjustToxLoss(0.5*volume)
 			if(show_message)
-				M << "<span class='notice'>You probably shouldn't have eaten that. Maybe you should of splashed it on, or applied a patch?</span>"
+				M << "<span class='notice'>You probably shouldn't have eaten that. Maybe you should have splashed it on, or applied a patch?</span>"
```

Comment:
yeah I also corrected that error because fuck that error

OMG BANDIT SNEAKING CHANGES INTO PRS WHEN WILL IT END